ado-build.yml

```diff
@@ -1,11 +1,24 @@
 pr:
 - 'main'
 
-pool:
-  vmImage: 'windows-latest'
-
 jobs:
 - job: NET_pipeline
+  strategy:
```

Comment:
consider adding "maxParallel" to reduce overall build time

Reply:
added